acs-integration-tests/src/test/java/com/ge/predix/cloud/foundry/application/DeleteApplications.java

```diff
@@ -0,0 +1,25 @@
+package com.ge.predix.cloud.foundry.application;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.testng.annotations.Test;
+
+import com.ge.predix.cloud.foundry.CloudFoundryUtilities;
+
+@ContextConfiguration("classpath:integration-test-spring-context.xml")
+public final class DeleteApplications extends AbstractTestNGSpringContextTests {
```

Comment:
This is deliberately pluralized since `US100676` deletes the asset adapter app as part of this class